src/metpy/io/metar.py

```diff
@@ -117,6 +119,7 @@ def parse_metar(metar_text, year, month, station_metadata=station_info):
     * 'date_time': Date and time of the observation, datetime object
     * 'wind_direction': Direction the wind is coming from, measured in degrees
     * 'wind_speed': Wind speed, measured in knots
+    * 'wind_gust': Wind gusts, measured in knots
```

Comment:
```suggestion
    * 'wind_gust': Wind gust, measured in knots
```

Below is without already

Reply:
Fixed.